fuel-vm/src/interpreter/debug.rs

```diff
@@ -47,12 +47,20 @@ where
 #[test]
 fn breakpoint_script() {
     use fuel_asm::op;
+    use fuel_tx::ConsensusParams;
+    use fuel_types::ChainId;
 
     let mut vm = Interpreter::with_memory_storage();
 
     let gas_limit = 1_000_000;
     let height = Default::default();
-    let params = ConsensusParameters::default();
+    let tx_params = TxParameters::default();
+    let predicate_params = PredicateParameters::default();
+    let script_params = ScriptParameters::default();
+    let contract_params = ContractParameters::default();
+    let fee_params = FeeParameters::default();
+    let chain_id = ChainId::default();
+    let gas_costs = GasCosts::default();
```

Comment:
It adds a lot of overhead=) if `ConsensusParams` works with values instead, you don't need to do it

Reply:
I missed this one. Adding now.